examples/observability/README.md

```diff
@@ -0,0 +1,69 @@
+# Sending Traces and Metrics to the OpenTelemetry Collector
+
+In this example, we will show you an example configuration for enabling the operator to send OpenTelemetry traces and metrics to the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector).
+The Collector will then be used to forward the gathered data to [Jaeger](https://www.jaegertracing.io) and [Prometheus](https://prometheus.io).
+
+## Prerequisites:
+
+This tutorial assumes that you have both Jaeger and the Prometheus Operator installed in your Cluster. 
+Also, please ensure that the Prometheus Operator has the required permissions to watch resources of the `keptn-lifecycle-controller-system` namespace (see https://prometheus-operator.dev/docs/kube/monitoring-other-namespaces/ as a reference).
+For setting up both Jaeger and Prometheus, please refer to their docs:
+
+- [Jaeger Setup](https://github.com/jaegertracing/jaeger-operator) 
+- [Prometheus Operator Setup](https://github.com/prometheus-operator/kube-prometheus/blob/main/docs/customizing.md)
+
+In this tutorial, we will assume that Jaeger will be installed in the `keptn-lifecycle-controller-system`, and the Jaeger collector endpoint is reachable under `http://jaeger-collector:14250`.
+
+## Configuring the OpenTelemetry Collector and Prometheus ServiceMonitor
+
+Once Jaeger and Prometheus are installed, you can deploy and configure the OpenTelemetry collector using the manifests in the `config` directory:
+
+```sh 
+kubectl apply -f config/
+```
+
+Also, please ensure that the `OTEL_COLLECTOR_URL` env vars of both the `klc-controller-manager`, 
+as well as the `keptn-scheduler` deployments are set appropriately. 
+By default, they are set to `otel-collector:4317`, which should be the correct value for this tutorial.
```

Comment:
this is problematic :/ 
The exporter is always created since it only checks the value of the env var. Having it by default `otel-collector:4317`, users will always get errors in the logs

Reply:
Created https://github.com/keptn-sandbox/lifecycle-controller/pull/150

Reply:
For simplicity I decided to set this per default to not require any modification of the operator deployment which should already be running in the cluster. Tests showed that the operator will not produce any error logs if that deployment is not reachable (other than the one log during the startup of the operator). For the scheduler, this will be fixed in #150